install.sh

```diff
@@ -92,6 +95,13 @@ tag_to_version() {
   # if version starts with 'v', remove it
   TAG="$REALTAG"
   VERSION=${TAG#v}
+
+  # Ensure script only installs fossa-cli less than v2.0.0
+  MAJOR_VERSION=$(printf %.1s "$VERSION")
```

Comment:
Why not use `grep` to ensure that the version matches `^v?1.*`?

Reply:
I ran this against shell-check, it gave warning for this: 

- https://github.com/koalaman/shellcheck/wiki/SC3015 (`[[ ]] is not POSIX sh compatible`)
- https://github.com/koalaman/shellcheck/wiki/SC3010 (`=~ is not POSIX sh compatible`)

Ref (using https://www.shellcheck.net/):
```
#!/bin/sh
set -e

if [[ "v2.3.1" =~ ^v1 ]]; then
  echo "v2.3.1 matches"
else
  echo "v2.3.1 does not match"
fi
```

Github CI uses dash, which also fails:
- https://github.com/meghfossa/mig-main/runs/3969969411?check_suite_focus=true
- https://github.com/meghfossa/mig-main/runs/3969975830?check_suite_focus=true

I'm leaning towards just using printf (it's part of coreutils) at this point. In our install script we use one of (wget || curl), I think there is a solid chance that coreutils will be accessible. Alpine distros uses busybox instead of gnu core utils, which also has printf and other expected utils (awk, grep, etc). Let me know what you think.

Reply:
Yeah, I think `printf` is fine. It'll work until we get to version 10, and this will be unnecessary long before that :)